server/src/domain/media/media.util.ts

```diff
@@ -0,0 +1,199 @@
+import { BitrateDistribution, TranscodeOptions, VideoCodecSWHandler, VideoStreamInfo } from '.';
+import { SystemConfigFFmpegDto } from '..';
```

Comment:
These types of imports can cause circular dependencies. Can you import from `./dto`, etc.